CITATION.bib

```diff
@@ -0,0 +1,6 @@
+@misc{Qiskit,
+    author = {{Qiskit contributors}},
+    title = {Qiskit: An Open-source Framework for Quantum Computing},
+    year = {2023},
```

Comment:
This comes from the original file in qiskit-metapacage, but why is the year 2023? and do we plan to update every year?

Reply:
I would expect this to match the version users cite -- so I would think we have to update it every year. But I also don't know what the common practice for this is 🙂

Reply:
I believe that the Zenodo DOI in the citation will point to the latest release, so the year of the most recent release should be correct in general, I think.